packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -439,9 +482,10 @@ public void setAccessibilityLiveRegion(@NonNull T view, @Nullable String liveReg
     }
   }
 
-  private static void setTransformProperty(@NonNull View view, ReadableArray transforms) {
+  private static void setTransformProperty(@NonNull View view, ReadableArray transforms, ReadableArray transformOrigin) {
```

Comment:
nit: transformOrigin should be marked as Nullable I think.

Reply:
Done!